xbmc/utils/LabelFormatter.cpp

```diff
@@ -451,6 +456,9 @@ void CLabelFormatter::FillMusicMaskContent(const char mask, const std::string &v
   case 'r': // userrating
     tag->SetUserrating(value[0]);
     break;
+  case 'b': // total discs
+    tag->SetTotalDiscs(atol(value.c_str()));
```

Comment:
Have you figured out when the LabelFormatter gets to set values?
FillMusicMaskContent is not called by anything.  Since this is not a song property I don't think it belong here either since the other values are all basic music file tag values (this method was probably once something to do with ripping CDs).

The whole label mask thing is a little ancient and odd. While you do need to add the rest for Disc total I don't think this  is wanted

Reply:
Yes, It does get called from `CFileItem::LoadMusicTag()`but only if there are no other tags found first & it's not a CD.  I doubt anyone is still using a 20+ year old method of naming their files with all the tag info in the filename,  but Kodi can do it if _really_ necessary !!  

> Yes, it could benefit from a redesign (in addition to doing far more in SQL not fileitem lists), but is a big job with high risk. Volunteering?

Erm....Nope !!  Don't think I'm quite feeling that brave at the moment !!